DuckDuckGo/CompleteDownloadRow.swift

```diff
@@ -71,7 +71,6 @@ struct CompleteDownloadRow: View {
         }
         .accessibilityLabel(UserText.actionShare)
         .buttonStyle(.plain)
-        .animation(nil)
```

Comment:
@miasma13 This API usage doesn't map neatly into the new SwiftUI animation API. I couldn't figure out what animation we were suppressing with it, since this action is showing a system Share sheet which will always animate. Let me know if you recall why this exists - I believe we could add it back, using the following code:

```
.animation(nil, value: UUID())
```

Reply:
Did a quick test of the download list with that line removed and didn't found any odd behavior. There is a slim chance that it was to suppress for some odd behaviour when run on old iOS (13 or 14 at that time).
I am fine with the removal.